framework/Sources/VGSFramework/Core/APIClient.swift

```diff
@@ -61,28 +62,25 @@ class APIClient {
         let path = baseURL.appendingPathComponent(path)
         // Fetch Request
         Alamofire.request(path,
-                          method: .post,
+                          method: method,
                           parameters: body,
-//                          encoding: DataEncoding.deferredToData,
                           headers: headers)
             .validate(statusCode: 200..<300)
             .responseJSON { response in
                 
                 switch response.result {
                 case .success:
-                    
-//                    let requestBody = response.request?.httpBody
-//                    let requestHeader = response.request?.allHTTPHeaderFields
-                    
-                    block(response.data, nil)
+                    let is200 = response.response?.statusCode == 200
```

Comment:
I think we dont need to add this check, since there is validation at top line: `.validate(statusCode: 200..<300)`

Reply:
You are right. But validation from 200 to 300 its validation for all cases to 300
for the success flag, I did this validation, just because 201 or 203 it's OK too.

Reply:
If it's extra checking unused then I should remove argument success form closure.